core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.spec.ts

```diff
@@ -35,11 +36,16 @@ describe('Translation Suggestion Review Modal Controller', function() {
   beforeEach(angular.mock.inject(function($injector, $controller) {
     contributionAndReviewService = $injector.get(
       'ContributionAndReviewService');
+    languageUtilService = $injector.get('LanguageUtilService');
+
     SiteAnalyticsService = $injector.get('SiteAnalyticsService');
     AlertsService = $injector.get('AlertsService');
     spyOn(
       SiteAnalyticsService,
       'registerContributorDashboardViewSuggestionForReview');
+    spyOn(
+      languageUtilService, 'getAudioLanguageDescription')
+      .and.returnValue('audio_language_description');
```

Comment:
This is added because `getAudioLanguageDescription` does not return a null value now. This function is used at component initialization